kraken/pod_scenarios/setup.py

```diff
@@ -34,3 +37,89 @@ def run(kubeconfig_path, scenarios_list, config, failed_post_scenarios, wait_dur
     except Exception as e:
         logging.error("Failed to run scenario: %s. Encountered the following " "exception: %s" % (pod_scenario[0], e))
     return failed_post_scenarios
+
+
+def container_run(kubeconfig_path, scenarios_list, config, failed_post_scenarios, wait_duration):
+    logging.info("scenario lis t" + str(scenarios_list))
```

Comment:
s/scenario lis t/scenario list

Reply:
This was just a comment for my debugging, removed it